gui/src/main/java/org/verapdf/cli/multithread/reports/MultiThreadProcessingHandler.java

```diff
@@ -0,0 +1,11 @@
+package org.verapdf.cli.multithread.reports;
+
+import org.verapdf.cli.multithread.BaseCliRunner;
+
+public interface MultiThreadProcessingHandler {
+	void startReport();
+
+	void fillReport(BaseCliRunner.ResultStructure result);
```

Comment:
Move `BaseCliRunner.ResultStructure` out of the runner to it's own definition, then it and this can be moved to https://github.com/veraPDF/veraPDF-library.